Online/Serialization/Serializer.Data.cs

```diff
@@ -944,6 +944,99 @@ public void SerializeNullable(ref WorldCoordinate? pos)
             }
 #if TRACING
             if (IsWriting) RainMeadow.Trace(this.Position - wasPos);
+#endif
+        }
+
+        public void Serialize(ref Dictionary<string, bool> data)
```

Comment:
@henpemaz, I added these.

Currently tested them on this commit by running local Story2P and ensuring Client received data correctly. Please optimize as necessary